apps/api/package.json

```diff
@@ -105,11 +105,14 @@
     "passport-github2": "^0.1.12",
     "passport-jwt": "^4.0.0",
     "passport-oauth2": "^1.6.1",
+    "pino": "^8.10.0",
+    "pino-http": "^8.3.3",
+    "pino-pretty": "^9.2.0",
```

Comment:
If we have moved everything to application generic, can we remove it from the API service?